spec/routing/publications_routing_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
```

Comment:
this was automatically generated with scaffold. Make sure you understand this test file also.